src/cio.ts

```diff
@@ -81,18 +87,34 @@ export async function identifyUserStreak(
   }
 }
 
-export async function identifyUser(
-  log: FastifyBaseLogger,
-  cio: TrackClient,
-  user: ChangeObject<User>,
-): Promise<void> {
+export async function identifyUser({
+  con,
+  log,
+  cio,
+  user,
+}: {
+  con: DataSource;
+  log: FastifyBaseLogger;
+  cio: TrackClient;
+  user: ChangeObject<User>;
+}): Promise<void> {
```

Comment:
Refactored to props syntax because I had to add `con`, we prefer it either way.

Reply:
Nice yeah we should just use object notation by default 🤦